etc/picongpu/draco-mpcdf/picongpu.profile.example

```diff
@@ -0,0 +1,68 @@
+# User Information ############################################################
+#   - edit those lines!
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="ALL"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Name and Path of this Script ################################################
+#   - do NOT change this line!
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# Text Editor for Tools #######################################################
+#   - edit those lines!
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# General Modules #############################################################
+#
+module purge
+
+module load git/2.14
+module load gcc/6.3
+module load cmake/3.7
+module load boost/gcc/1.64
+module load impi/2017.3
+module load hdf5-mpi/gcc/1.8.18
+
+# Other Software ##############################################################
+#
+# needs to be compiled by the user
+export PNG_ROOT=$HOME/lib/libpng-1.6.32
+export PNGWRITER_ROOT=$HOME/lib/pngwriter-0.6.0
+export SPLASH_ROOT=$HOME/lib/splash-1.6.0
+
+export LD_LIBRARY_PATH=$PNG_ROOT/lib:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=$PNGWRITER_ROOT/lib:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=$SPLASH_ROOT/lib:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=$BOOST_HOME/lib:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=$HDF5_HOME/lib:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=$I_MPI_ROOT/lib64:$LD_LIBRARY_PATH
+
+export HDF5_ROOT=$HDF5_HOME
+
+export CXX=$(which g++)
+export CC=$(which gcc)
+
+# PIConGPU Helper Variables ###################################################
+#
+export PICSRC=$HOME/src/picongpu
+export PIC_BACKEND="omp2b"
```

Comment:
add architecture again? `:haswell` did not work in our first tests, but this could have been due to an too old gcc in the beginning.

Reply:
Yes please add it.

Reply:
did it work now (with the right version of gcc after setting `$CXX`?)